TECHNICAL.md

```diff
@@ -1,7 +1,7 @@
 # Technical Reference
-This document describes how MetaSWAP and MODFLOW are coupled.
+This document describes how MetaSWAP and MODFLOW are coupled. It is intended for groundwater modellers, who need to know which variables are exchanged between modelcodes and at which moment. For details of the inner workings of the code, we refer to the docstrings in the code.
```

Comment:
"numerical kernels" might fit better than "modelcodes"?

Sidenote: I recommend to start a newline after every sentence for markdown or latex documents.
The output does not change, but the diff becomes easier to read.